docs/community_guide.md

```diff
@@ -0,0 +1,39 @@
+# Community Guide
+
+Welcome to the PROJECTNAME Community Guide!
+
+> If your project is not going to have a docsite, the following information should be present in your project README as in the [README template](https://github.com/ansible-community/project-template/blob/main/README.md).
+
+## Code of Conduct
+
+> If your project does not belong to Red Hat GitHub orgs, refer to a CoC violation complaint raising mechanism relevant to your project.
```

Comment:
I'm unsure about this line. This is specifically for ansible projects and we require all ansible projects to have the Ansible CoC. AFAIK only collections get the option of having a different CoC.